core/src/scene/sceneLoader.cpp

```diff
@@ -64,6 +65,97 @@ void printFilters(const SceneLayer& layer, int indent){
     }
 };
 
+void SceneLoader::applyGlobalProperties(Node& node, Scene& scene) {
+    switch(node.Type()) {
+    case NodeType::Scalar:
+        {
+            std::string key = node.Scalar();
+            const std::string wildcard = "_$_";
```

Comment:
I think elsewhere we use `:` as a nested-value-delimiter. Think it would make sense to define a delimiter globally for this file?